app/services/identity_linker.rb

```diff
@@ -40,10 +41,10 @@ def link_identity(
 
   private
 
-  def process_ial(ial)
+  def process_ial(ial, include_identity_session_attributes:)
     @ial = ial
     now = Time.zone.now
-    process_ial_at(now)
+    process_ial_at(now) if include_identity_session_attributes
```

Comment:
in hindsight, it's frustrating that these `ial_at` columns were added to this table, and updated in this way because they make this all kind of complicated